src/runtime/server/handler-graphql.ts

```diff
@@ -81,6 +82,48 @@ export const createRequestHandlerGraphQL: CreateHandler = (schema, createContext
   return sendSuccess(res, result)
 }
 
+export type ExecuteGraphQl = (
+  query: string,
+  variables: Record<string, unknown> | null,
+  operation: string | null
+) => Promise<ExecutionResult>
+
+export type CreateGraphQLExecutor = (schema: GraphQLSchema, context: ContextCreator) => ExecuteGraphQl
+
+export const createGraphQLExecutor: CreateGraphQLExecutor = (schema, contextCreator) => async (
+  query,
+  variables,
+  operation
+) => {
+  const source = new Source(query)
+  const context = await contextCreator({})
```

Comment:
Hm, how is this going to work out?

AFAICT it requires that no resolvers in your app rely on request object data.

If any do, there could be an error not caught until runtime.

This is the main design problem here to discuss I think.